rockcraft.yaml

```diff
@@ -12,7 +12,7 @@ platforms:
 cmd: ["lte", "-run_echo_server=true", "-logtostderr=true", "-v=0"]
 
 env:
-  - DATABASE_SOURCE: f"dbname=magma user=username password=password host=127.0.0.1 sslmode=disable"
+  - DATABASE_SOURCE: "dbname=magma user=username password=password host=172.17.0.2 sslmode=disable"
```

Comment:
I'd keep this to `127.0.0.1`, it's strange for it to be a very specific address

Reply:
Changed.